coop_print_badge/__manifest__.py

```diff
@@ -1,36 +1,35 @@
-# -*- coding: utf-8 -*-
 # Copyright (C) 2017-Today: La Louve (<http://www.lalouve.net/>)
 # @author: Julien Weste (julien.weste@akretion.com)
+# Copyright (C) 2019-Today: Druidoo (<https://www.druidoo.io>)
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
-
 {
     'name': 'Coop - Print Badge',
-    'version': '9.0.1.0.0',
+    'version': '12.0.1.0.0',
     'category': 'Custom',
     'summary': 'Print partner\'s badge',
     'author': 'La Louve',
     'website': 'http://www.lalouve.net',
     'depends': [
         'base',
-        'barcodes_generate',
+        'web',
+        'barcodes_generator_abstract',
+        'barcodes_generator_partner',
+        'barcodes_generator_product',
         'coop_shift',
         'coop_membership',
-        'report',
+        'coop_capital_certificate',
```

Comment:
Why are we adding this `coop_capital_certificate` dependancy? :thinking: 
Is this intentional?

Reply:
that module's action is use in our module on this line, https://github.com/druidoo/FoodCoops/blob/12.0-mig-coop_print_badge/coop_print_badge/views/actions.xml#L73

Reply:
Thanks for clarifiying!